src/org/jitsi/impl/neomedia/rtp/MediaStreamTrackDesc.java

```diff
@@ -29,23 +31,47 @@
 public class MediaStreamTrackDesc
 {
     /**
+     * The system property name that holds the minimum time (in millis) that is
+     * required for the media engine to generate a new key frame.
```

Comment:
The max frequency that the webrtc engine generates a key frame is 300 ms. We take that into account to identify batches of key frames which helps us identify which simulcast flows are active/inactive.